mpeg2/src/muxer.rs

```diff
@@ -196,6 +190,7 @@ pub struct Packet<'a> {
     pub random_access_indicator: bool,
     pub pts_90khz: Option<u64>,
     pub dts_90khz: Option<u64>,
+    pub temi: Vec<TEMITimelineDescriptor>,
```

Comment:
I'm always a fan of spelling things out fully in `pub` APIs:

```suggestion
    pub temi_timeline_descriptor: Vec<TEMITimelineDescriptor>,
```

Reply:
will keep that in mind!!!!